test/annexB/built-ins/Date/prototype/getYear/not-a-constructor.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2020 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-ecmascript-standard-built-in-objects
+description: >
+  Date.prototype.getYear does not implement [[Construct]]
+info: |
+  ECMAScript Function Objects
+
+  Built-in function objects that are not identified as constructors do not
+  implement the [[Construct]] internal method unless otherwise specified in
+  the description of a particular function.
+includes: [isConstructor.js]
+features: [Reflect.construct, arrow-function]
+---*/
+
+assert.sameValue(
+  isConstructor(Date.prototype.getYear),
+  false,
+  'isConstructor(Date.prototype.getYear) must return false'
+);
+
+assert.throws(TypeError, () => {
+  let date = new Date(Date.now()); new date.getYear();
+}, '`let date = new Date(Date.now()); new date.getYear()` throws TypeError');
```

Comment:
these assertions are "fine" but i don't see what value they add, and this seems like the wrong place to put them - they don't test what the file is claiming to test.

Reply:
I think maybe you're missing the context for why these tests are even coming into existence? When we were actively capturing JSC tests, for the purpose of curating novel material, or finding gaps in existing coverage, https://github.com/tc39/test262/issues/1739 was filed to call out this test: https://github.com/WebKit/webkit/blob/master/JSTests/stress/builtin-function-is-construct-type-none.js. 

I've added further specification citations to the info for these tests, citing the step in `EvaluateNew ( constructExpr, arguments )`.

Reply:
I was missing the context; however, the test files say they're testing the lack of a [[Construct]] slot, which is achieved by the first test.

Reply:
Got it, and yes—that's what I thought you were referring to and what I hoped to address with: 

> I've added further specification citations to the info for these tests, citing the step in EvaluateNew ( constructExpr, arguments ).

Reply:
I thought I had added that language to all of them, looks like I missed some :| Fixing that now.